src/app/xm.module.ts

```
@@ -62,6 +62,18 @@ const paginatorOptions: MatPaginatorDefaultOptions = {
     formFieldAppearance: 'outline',
 };
 
+export const ngxMaskConfig = (): IConfig | object => {
+    return {
+        ...(initialConfig ?? {}),
+        patterns: {
+            ...(initialConfig.patterns ?? {}),
+            'C': {
+                pattern: new RegExp('[а-яА-Я]'),
```

maybe better by default have a-zA-Z

---

We have extended the default ngx-mask, so by default, it includes the characters a-zA-Z